src/components/CalendarGrid.vue

```diff
@@ -220,6 +220,19 @@ export default {
 			}
 		}, 1000)
 
+		// Trigger the select event programmatically on initial page load to show the new event
+		// in the grid.
+		this.$nextTick(() => {
```

Comment:
Using `await this.$nextTick();` instead of the callback *should* work.

Reply:
Neat! I didn't know that it returns a promise.